techniques/general/G209.html

```diff
@@ -2,150 +2,126 @@
-                     segments. .
-                  </figcaption>
+                  <figcaption>図2 円グラフのセグメント間のコントラストは1.1〜1.8の範囲である。セグメントを識別するため、3:1以上のコントラスト比の色を持った線が追加されている。</figcaption>
                   
                </figure>
             </section>
             <section class="example" id="example-3-map-with-border-boundaries">
-               <h3>Example 3: Map with border boundaries</h3>
+               <h3>事例3: 境界線がある地図</h3>
                <figure id="figure-map-border-boundaries">
                   <img src="img/map-with-borders.png" alt="Map showing simple regions with dark background colours and a light border between regions." />
                   
-                  <figcaption>Figure 3The color contrast of areas within the map range between 1.3 and 1.8. A boundary line
-                     is added with a color contrast ratio of at least 3:1 with the area colors.
-                  </figcaption>
+                  <figcaption>図3 地図内の領域の色のコントラストは1.3〜1.8の範囲である。境界線は、領域の色に対して3:1以上のカラーコントラスト比で追加される。</figcaption>
                   
                </figure>
             </section>
             <section class="example" id="example-4-pie-chart-with-black-and-white-borders">
-               <h3>Example 4: Pie chart with black and white borders</h3>
+               <h3>事例4: 黒と白の線がある円グラフ</h3>
                <figure id="figure-pie-black-white">
                   <img src="img/pie-chart-contrast-two-color-borders.png" alt="Pie chart of favorite candy flavours, including text labels and black and white borders between segments." />
                   
-                  <figcaption>Figure 4The colored areas in the chart are both dark and light. A black and white boundary
-                     line is added to ensure sufficient contrast between both light and dark colors.
-                  </figcaption>
+                  <figcaption>図4 グラフの色付きの領域には、暗いものと明るいものの両方がある。明るい色と暗い色の両方の間に十分なコントラストを確保するため、黒と白の境界線が追加される。</figcaption>
                   
                </figure>
             </section>
          </section>
          <section id="resources">
-            <h2>Resources</h2>
-            <p>Resources are for information purposes only, no endorsement implied.</p>
+            <h2>関連リソース</h2>
+            <p>リソースは、情報提供のみを目的としており、推奨を意味するものではない。</p>
             <ul>
                
-               <li><a href="../../understanding/contrast-minimum.html#resources">Understanding SC 1.4.3 Contrast (minimum)</a> lists color contrast tools.
-               </li>
+               <li><a href="../../understanding/contrast-minimum.html#resources">達成基準 1.4.3: コントラスト (最低限)を理解する</a> はカラーコントラストツールの一覧である。</li>
                
             </ul>
          </section>
          <section id="related">
-            <h2>Related Techniques</h2>
+            <h2>関連する達成方法</h2>
             <ul>
                				
                <li><a href="https://w3c.github.io/wcag/techniques//G207.html">G207.html: </a></li>
```

Comment:
```suggestion
               <li><a href="G207.html">G207.html</a></li>
```

非常に不細工ですが、とりあえずこう変更しておきますか…